src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html

```diff
@@ -49,7 +50,7 @@
   (scrolled)="onScroll()"
   [scrollWindow]="false"
 >
-  <table *ngIf="!isLoading; else spinner" mat-table [dataSource]="dataSource" matSort class="mat-elevation-z0" cdkDropListGroup>
+  <table *ngIf="!isLoading; else spinner" mat-table [dataSource]="dataSource" matSort class="mat-elevation-z0 table" cdkDropListGroup>
```

Comment:
Add "th" headers to this table.

Reply:
Add a description to this table.

Reply:
https://sonarcloud.io/project/issues?id=ita-social-projects-green-city-client&open=AXroheaMtOsjBe3j_bEV&pullRequest=1004&resolved=false&types=BUG

Reply:
this bug fixed by adding role="presentation" in bugfix/#3012 pull request